bin/waagent2.0

```diff
@@ -3300,6 +3300,87 @@ class Certificates(object):
             filename = str(index) + ".prv"
         return self
 
+
+class RemoteAccess(object):
```

Comment:
This file is not used, and shouldn't be edited.

Reply:
done